lighty-core/lighty-controller-guice-di/src/main/java/io/lighty/core/controller/guice/LightyControllerModule.java

```diff
@@ -70,8 +71,8 @@ protected void configure() {
                 .toInstance(lightyServices.getDOMSchemaService());
         bind(DOMYangTextSourceProvider.class)
                 .toInstance(lightyServices.getDOMYangTextSourceProvider());
-        bind(DOMNotificationSubscriptionListenerRegistry.class)
-                .toInstance(lightyServices.getDOMNotificationSubscriptionListenerRegistry());
+        bind(DOMNotificationRouter.class)
```

Comment:
DOMNotificationPublishDemandExtension can not be used since DOMNotificationRouter does not implement it as a interface.
Please see this change for more info:
https://github.com/opendaylight/mdsal/commit/761ac571ff8ebc85885e78cd4e07f48cc3f3888e#diff-9fc51f89142dcca65f1645a9c019eb5e227f57400a3859fc7fe1e99f8f5edc34

Reply:
ACK